src/cluster_asm.c

```diff
@@ -962,6 +993,12 @@ void asmRdbChannelSyncWithSource(connection *conn) {
         /* The destination node did not reply */
         if (err == NULL) goto no_response_error;
 
+        /* Ignore `\n` that may be sent to keep the connection alive */
+        if (sdslen(err) == 0) {
+            sdsfree(err);
```

Comment:
Maybe we can add a log line. While investigating the logs, it makes it clear why replication has not started yet.

Reply:
make sense, let me add a debug log